storio-content-resolver/src/test/java/com/pushtorefresh/storio/contentresolver/integration/IntegrationTest.java

```diff
@@ -307,4 +312,32 @@ public void call(Changes changes) {
 
         return storedTweets;
     }
+
+    @NonNull
+    PreparedGetCursor createQueryWithNullResult() {
```

Comment:
Why is it Integration Test? I guess it's more likely Unit test, you mocked everything here :) Our Integration Tests are integration because we use Robolectric to emulate Android Emulator's behavior.